packages/office-ui-fabric-react/src/components/Button/examples/Button.CommandBar.Example.tsx

```diff
@@ -1,34 +1,47 @@
 import * as React from 'react';
-import { CommandBarButton, IButtonProps } from 'office-ui-fabric-react/lib/Button';
+import { CommandBarButton, IContextualMenuProps, IIconProps, Stack, IStackStyles } from 'office-ui-fabric-react';
 
-export const ButtonCommandBarExample: React.FunctionComponent<IButtonProps> = props => {
+export interface IButtonExampleProps {
+  // These are set based on the toggles shown above the examples (not needed in real code)
+  disabled?: boolean;
+  checked?: boolean;
+}
+
+const menuProps: IContextualMenuProps = {
+  items: [
+    {
+      key: 'emailMessage',
+      text: 'Email message',
+      iconProps: { iconName: 'Mail' }
+    },
+    {
+      key: 'calendarEvent',
+      text: 'Calendar event',
+      iconProps: { iconName: 'Calendar' }
+    }
+  ]
+};
+const addIcon: IIconProps = { iconName: 'Add' };
+const mailIcon: IIconProps = { iconName: 'Mail' };
+const stackStyles: Partial<IStackStyles> = { root: { height: 44 } };
```

Comment:
Any reason why we have to specify `Partial` here instead of just `IStackStyles`?

Reply:
I got in the habit of doing this since some style interfaces have required members, though it may not actually be needed here.